### DISCUSSION TITLE
Ability to Match Multiple Dependency Versions to Single Dependency?

### DISCUSSION THREAD

**Discussion by TheNathanBlake** [npm/rfcs#758] [Question]
Hey so I've tried looking through documentation and Googling for a way to do this and my search came up dry, but I've run into issues around this for certain library bundles, Parcel in particular, where certain sub-modules are dependent on a particular main module version being used.

Is there a way in the package.json to tell a particular dependency to use the same version number as another dependency?  E.g.:

```
"devDependencies": {
"parcel": "^2.11.3",
"@parcel/resolver-default": "parcel" <- whatever key string would control this
```

**Comment by ljharb** [comment]
Given that locking version numbers across packages is often considered a bad practice (many of the projects that have done so have stopped, because it forces needless version bumps), I wouldn't expect an automated way to do this.
